src/components/ewt-littlefs-manager.ts

```
@@ -0,0 +1,837 @@
+      this._flashProgress = 0;
+    }
+  }
+
+  private _cleanup() {
+    if (this._fs) {
+      try {
+        // Don't call destroy() - just let garbage collection handle it
+      } catch (e) {
+        console.error("Error cleaning up LittleFS:", e);
+      }
+      this._fs = null;
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)